### PR TITLE
Improve overall `bundles` documentation

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Add missing information
- Shorten the instructions
- Improve the UX when reading the bundles docs
- Fix the links
- Improve the overall documentation

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
+# Releases
+
 This document describes how the release flow looks like and how to create a bundle release.
 
 ## Release process
 
-Every change in the master branch which passes all checks triggers creating a `latest` tag and release.
+Every change in the master branch which passes all checks triggers the `latest` tag and release.
```
Comment (mmitoraj) [inline docs/releases.md]: triggers or creates the `latest` tag? Maybe creates the `latest` tag and triggers a release?

File: docs/getting-started.md [hunk 2]
```diff
@@ -1,76 +1,60 @@
...
+This document describes how to configure your forked Helm Broker `bundles` repository.
 
 ## Prerequisites
 
-Before you start, install:
-
-* [Kyma](https://kyma-project.io/docs/latest/root/kyma#tutorials-tutorials) in version v0.4.1 or later
-* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in version v1.10.0 or later
-* [svcat CLI](https://svc-cat.io/docs/cli/) in version v1.10.0 or later
+* [Kyma](https://kyma-project.io/docs/latest/root/kyma#tutorials-tutorials) in version v0.4.1 or higher
```
Comment (mmitoraj) [inline docs/getting-started.md]: Shouldn't we link to the Kyma installation section instead of the tutorial?

Comment (klaudiagrz) [reply docs/getting-started.md]: Good point, haven't noticed that, thanks! :)

File: docs/releases.md (hunk 1 above)
Comment (klaudiagrz) [reply docs/releases.md]: `creates the latest tag and triggers a release` option is good

**Pull request merged**